src/EasyAbp.Abp.SettingUi.Web/Pages/SettingManagement/SettingUi/Index.js

```diff
@@ -46,23 +46,35 @@
         })
     }
 
-    $('#tabs-nav .nav-item .nav-link').click(function () {
-        var _this = $(this);
-        if (_this.attr('data-bs-target') !== undefined) {
-            return;
-        }
+    var bind_nav_event_fn = function (element) {
         var btn_set = setInterval(function () {
-            var _id = '#' + _this.data('id') + " ";
+            var _id = '#' + element.data('id') + " ";
             event_fn(_id);
             if ($(_id + " form").length > 0) {
```

Comment:
When no form exists, we can consider skipping the next steps. 
`if ($(_id + " form").length == 0) return`

Reply:
If form is not setting ui's form. We also need to clear the interval.